### PR TITLE
Documentation and Guidelines Update

### PR DESCRIPTION
**Pull request by mdlewisfb**
# Description
Pull Request template is missing, as are code of conduct and contributing guidelines. Proposing a pull request template here, can also add the other two or save those for a separate PR.
Updated the README.md as some of the links were broken, also wanted to add the link to the discussion forum.

# Test Plan
N/A as no code changes, just need to verify that once this is merged, new pull requests are autofilled with the template.

# Issues
#178

### CODE REVIEW

**Comment by mdlewisfb** [comment]
Need clarity on the code of conduct and contributing guidelines, as I would like to include those here. @deeliminator @markhor @jspectacular any opinions on what should be included?

**Comment by mdlewisfb** [comment]
Did an update that just moved the pr_template up one level, as I found out that when using a PULL_REQUEST_TEMPLATE directory GitHub will not autofill new pull requests with the template, but will do so if the template is located directly in .github/

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Code of Conduct
+
+Facebook has adopted a Code of Conduct that we expect project participants to adhere to.
```
Comment (markhor) [inline CODE_OF_CONDUCT.md]: This should be TIP or OC

**Comment by markhor** [comment]
@mdlewisfb, Some minor feedback on the "code of conduct". Can you please address it.

**Comment by deeliminator** [comment]
Looks good.

**Pull request merged**